saas/api/metrics.py

```diff
@@ -60,6 +63,40 @@ def get(self, request, *args, **kwargs): #pylint: disable=unused-argument
         return Response(result)
 
 
+class RevenueAPIView(ProviderMixin, APIView):
+    """
+    Produce revenue stats
+    """
+    def get(self, request, *args, **kwargs):
+        ends_at = request.GET.get('ends_at', None)
+        if ends_at:
+            ends_at = parse_datetime(ends_at)
+        ends_at = datetime_or_now(ends_at)
+
+        table_key = request.GET['table_key']
```

Comment:
See this commit: https://github.com/marmida/djaodjin-saas/commit/b6f613c5dfbba1a70d219a754a7e8bef59ffb73b

I also emailed a patch to the private repo.

This is a hacky solution, because I need to get a URL through Javascript, but cannot materialize a URL without having all its arguments, so I use a bogus placeholder value.